low-level/files/opt/seagate/sspl/bin/sspl_provisioner_init

```diff
@@ -139,7 +138,7 @@ start_req_services(){
         echo "Changing SSPL state to 'active', disable step once HA integration complete"
         echo "*************************************************************************"
         echo "state=active" > /var/$PRODUCT_FAMILY/sspl/data/state.txt
-        PID=`/sbin/pidof -s /usr/bin/sspl_ll_d`
+        PID=`/usr/bin/pgrep -d " " -f /usr/bin/sspl_ll_d`
```

Comment:
single latest pid needs to be picked up

Reply:
Same comment as above. what if we kill all PID's which is related to sspl_ll_d

Reply:
i dont think kill command can take multiple PIDs

Reply:
Kill command have the capability to take multiple PIDs.
https://www.thegeekdiary.com/how-to-kill-processes-in-linux-using-kill-killall-and-pkill/#:~:text=The%20root%20user%20can%20use,on%20a%20single%20command%20line.

Reply:
Will there be multiple PIDs after removing pyinstaller?

Reply:
No. only one PID is there.

Reply:
`sudo systemctl --kill-who=main --signal=SIGHUP kill sspl-ll`

You can use systemctl to send SIGHUP to SSPL without finding PIDs